ajax/libs/jquery.sumoselect/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery.sumoselect",
+  "version": "3.0.2",
+  "description": "A jQuery plugin that progressively enhances an HTML Select Box into a Single/Multiple option dropdown list",
```

Comment:
@clairetsai818 Why not use `description` and `keywords` from upstream?
ref: **description**: https://github.com/HemantNegi/jquery.sumoselect/blob/v3.0.2/jquery.sumoselect.jquery.json#L5
**keywords**: https://github.com/HemantNegi/jquery.sumoselect/blob/v3.0.2/jquery.sumoselect.jquery.json#L6-L22